_posts/2021-04-13-upgrade-notification-for-ase.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Upgrade Notifications for App Service"
+author_name: "Michimune Kohno and Sanghmitra Gite"
+toc: true
+toc_sticky: true
+tags:
+    - Upgrade notifications
+    - Azure Monitor
+    - App Service Environment
+    - Logic Apps
+---
+
+We are happy to announce that the upgrade notifications for App Service Environment is available. With an upgrade notification you will be able to receive email, SMS text, or take action before a platform upgrade actually starts.
```

Comment:
I assume this is still in preview, yeah? If so, let's please update this to make it clear rather than just saying available.

Reply:
I see. Yes, it's in preview. I will update the doc.

Reply:
Updated